app/scripts/metamask-controller.actions.test.js

```diff
@@ -96,6 +97,17 @@ describe('MetaMaskController', function () {
         ]),
       );
     metamaskController = new MetaMaskController({
+      initState: {
```

Comment:
The default network for NetworkController is now Mainnet, but these tests assumed that it is localhost.

Reply:
We can make the default network whatever we want, since we set the initial state. Could we make this behave the same as before?